source/menu-template.ts

```diff
@@ -268,7 +268,7 @@ export class MenuTemplate<Context> {
 	 * })
 	 * submenuTemplate.manualRow(createBackMainMenuButtons())
 	 *
-	 * menuTemplate.submenu('enter submenu', 'unique', submenuTemplate)
+	 * menuTemplate.submenu('unique', submenuTemplate, { text: 'Text' })
```

Comment:
Having the text there helps with understanding what the button does.
```suggestion
	 * menuTemplate.submenu('unique', submenuTemplate, { text: 'enter submenu' })
```